console/src/components/Services/Data/constants.js

```diff
@@ -45,3 +45,176 @@ export const Reals = ['float4', 'float8', 'numeric'];
 export const Numerics = [...Integers, ...Reals];
 
 export const defaultDataTypeToCast = 'text';
+
+export const ERROR_CODES = {
+  postgresError: {
+    ERRORCODE: 'postgres-error',
+    ERRORS: [
+      'Not-NULL violation. null value in column <column-name> violates not-null constraint',
+      'query execution failed',
+      'postgres transaction error',
+      'connection error',
+      'postgres query error',
+    ],
+    HTTPCODE: '500',
+  },
+  permissionDenied: {
+    ERRORCODE: 'permission-denied',
+    ERRORS: [
+      'select on <column/table> for role <role-name> is not allowed.',
+      '<permission-type> on <table-name> for role <role-name> is not allowed',
+      'role <role-name> does not have permission to <permission-type> column <column-name>',
+    ],
+    HTTPCODE: '400',
+  },
+  notExists: {
+    ERRORCODE: 'not-exists',
```

Comment:
`code` for simplicity. Also, let's stick to either upper case or lower case. This isn't comfortable to read:
```
ERROR_CODES.dependencyError.ERRORCODE
```